test/syntax_tree.jl

```diff
@@ -28,7 +28,6 @@
         "(call-i (call-i a::Identifier *::Identifier b::Identifier) +::Identifier c::Identifier)"
 
     @test sprint(highlight, t[1][3]) == "a*b + c\n# ╙"
-    @test sprint(highlight, t.source, t.raw, 1, 3) == "a*b + c\n# ╙"
```

Comment:
why is this test removed?

Reply:
Reaches into the internals of SyntaxNode which changed. The line prior shows that the SyntaxNode itself can still be highlighted with the same result.